makeflow/src/makeflow.c

```diff
@@ -975,7 +970,7 @@ int main(int argc, char *argv[])
 	}
 	s = getenv("WORK_QUEUE_FAST_ABORT_MULTIPLIER");
 	if(s) {
-		wq_option_fast_abort_multiplier = atof(s);
+		wq_option_fast_abort_multiplier = xxstrdup(s);
```

Comment:
There is batch_queue_set_int_option. No need to change the type of wq_option_fast_abort_multiplier. I rather have ints as ints.